### ISSUE TITLE
Update framework and dual monitor support required

### ISSUE THREAD

**Issue by yoncu** [KennethGomez/LiveShot#13]
It should work on current systems without the need for additional framework installation and there is no dual monitor support. These updates will be very useful.

**Comment by KennethGomez** [comment]
Yes, you're completely right, being a tool for Screenshots, it shouldn't be required something like a framework. I am aware of the dual monitor lacking but actually, this project is not maintained for now. I want to do this project again on another language/framework that allows LiveShot to be used without any external framework, maybe using Rust or C++.

Thank you very much for your feedback!

**Comment by yoncu** [comment]
We met earlier on google talk. If you add these supports, we may pay you some fee.

**Comment by remy-burney-pf** [comment]
Just use [LightShot ](https://app.prntscr.com/)

**Comment by yoncu** [comment]
I need your support. Open source codes can be your own, you can share them on githup. But it requires up-to-date framework and dual monitor support.
I would be very happy if you could help me with this. I opened the website and integrated the application, but there are many problems without an update. It is not liked. Not used.
Please review the site and app: http://f8.re/
If you want, I can send you the latest source codes or I can give you the passwords of a windows system login ip address with extended access for your work.

**Comment by mtsbh** [comment]
> Just use [LightShot ](https://app.prntscr.com/)

LightShot gives me error when trying to download and I found this. Is there future in this project? Because I can't even launch it, it requires NET and wouldn't download it by itself. And I have .NET last version. Thats the problem.
